strawberryfields/gbs/data.py

```diff
@@ -461,13 +461,22 @@ def T(self) -> bool:
 class Formic(MoleculeDataset):
     """Zero temperature formic acid.
 
-    TODO Cite sources for w, wp, Ud, d + include visualisation
+    The molecular parameters are from :cite:`Huh_2015`.
+
+    **Molecule:**
+
+    .. |formic| image:: ../../_static/formic.png
+        :align: middle
+        :width: 250px
+        :target: javascript:void(0);
+
+    |formic|
 
     Attributes:
         n_mean = 1.56
         threshold = False
-        n_samples = 100000

```

Comment:
Did you generate new data? Could we just go through the original data and pad with zeros?

Reply:
I generated new data mainly to check everything works fine after upgrading the functions for finite T. Yes we can just pad the old data with zeros. In any case, we do not really need more than 20000 samples to get convergence in the shape of the spectrum.